streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamsMetricsImplTest.java

```diff
@@ -70,44 +60,40 @@ public void testRemoveSensor() {
 
     @Test
     public void testLatencyMetrics() {
-        final String groupName = "doesNotMatter";
+        final StreamsMetricsImpl streamsMetrics = new StreamsMetricsImpl(new Metrics(), "");
+        final int defaultMetrics = streamsMetrics.metrics().size();
```

Comment:
The skipped records metrics are now always present. Rather than updating the hard-coded value, I did this to make the test less brittle.